server/drivers/presto/index.js

```diff
@@ -34,8 +34,9 @@ function runQuery(query, connection) {
   let incomplete = false;
   const rows = [];
   const port = connection.port || 8080;
+  const protocol = connection.useHTTPS ? 'https' : 'http';
   const prestoConfig = {
-    url: `${connection.host}:${port}`,
+    url: `${protocol}://${connection.host}:${port}`,
```

Comment:
I think this could break existing connections since they may specify protocol in host field. Could you check and/or strip `http`/`https` from `connection.host`?

Reply:
OK. I modify the code to check the `host`. If `host` don't have `http`/`https`, then add the protocol.